src/main/java/deanery/Visit.java

```diff
@@ -0,0 +1,30 @@
+package deanery;
+
+import deanery.exception.WrongHour;
+import deanery.exception.WrongNumber;
+import deanery.room.Room;
+import deanery.student.Student;
+
+import java.util.List;
+
+
+public class Visit {
+    static List<Room> rooms = RoomData.createRooms();

```

Comment:
generally doing an action (so creating a list of rooms) during the class creation (because it happens before any method is called) is not the best practice -> you couldn't handle any exceptions this way and the program would not start